test/test_torch.py

```diff
@@ -1439,6 +1439,24 @@ def test_nondeterministic_alert_interpolate_bilinear(self, device):
             'upsample_bilinear2d_backward_out_cuda',
             torch.device(device).type == 'cuda')
 
+    @skipIfTorchInductor("aot-autograd issue")
```

Comment:
🤔 what broke?

Reply:
All of these tests are skipped this way. What broke was requires_grad was not set correctly on some of the tensors. TBH, it is probably worth figuring out what went wrong; maybe it's "requires_grad tensors was created inside graph"